src/main/java/models/Person.java

```diff
@@ -0,0 +1,40 @@
+package main.java.models;
+
+public abstract class Person {
+    private String name;
+    private String birthCity;
+    private String gender;
+
+    public Person() {
+    }
+
+    public Person(String name, String birthCity, String gender) {
```

Comment:
Métodos com mais de uma linha de código tu também adiciona uma linha vazia para ficar melhor de visualizar o código.

Reply:
Até se for um construtor?

Reply:
yeah